src/components/FileUploader/FileUploader.js

```diff
@@ -125,7 +125,7 @@ export class Filename extends Component {
       return (
         <div
           className="bx--loading"
-          style={Object.assign(style, { width: '1rem', height: '1rem' })}
+          style={{ ...style, ...{ width: '1rem', height: '1rem' } }}
```

Comment:
Changed this to a spread operator because of some issue I was having in tests with the Object.assign running into read only properties.

Let me know if I should switch it back and work through the issues.

Reply:
Do we need the second spread on the object literal with width/height? Seems like we'd just spread style and then add those attributes directly, unless I'm missing something.

Reply:
Nope, I'm just being dumb 😄